scripts/hostapd/install

```diff
@@ -32,15 +32,18 @@ sudo sed -i 's/#net.ipv6.conf.all.forwarding=1/net.ipv6.conf.all.forwarding=1/'
 
 # Get network name and password
 APSSID=$(sudo grep -m 1 '"ipv6"' /etc/cjdroute.conf | awk '{ print $2 }' | sed 's/[",]//g' | sed "s/.*:/$MESH_NAME-/g")
-while [ "${#APPASS}" -lt 8 ] || [ "${#APPASS}" -gt 63 ]; do
-    read -p "Set WPA2-PSK password for WiFi Access Point $APSSID (8-63 characters): " APPASS;
-done
 
 # Configure network with hostapd
 sudo cp "$BASE_DIR/nat.sh" /etc/hostapd/nat.sh
 sudo cp "$BASE_DIR/hostapd.conf" /etc/hostapd/hostapd.conf
+sudo cp "$BASE_DIR/hostapd.eap_user" /etc/hostapd/hostapd.eap_user
+
+/bin/bash $BASE_DIR/certs/bootstrap $BASE_DIR/certs
+
+sudo cp "$BASE_DIR/certs/ca.pem" /etc/hostapd/ca.pem
+sudo cp "$BASE_DIR/certs/server.pem" /etc/hostapd/server.pem
+
 sudo echo "ssid=$APSSID" | sudo tee --append /etc/hostapd/hostapd.conf > /dev/null
-sudo echo "wpa_passphrase=$APPASS" | sudo tee --append /etc/hostapd/hostapd.conf > /dev/null
```

Comment:
@pierrebeaucamp suggested a configuration where any username and password is accepted. Maybe we can default to that if no a blank password is entered in the prompt? Something like:

```
read -p "Set WPA2-EAP password for WiFi Access Point $APSSID (8-63 characters, or blank to accept any username and password): " APPASS;
```